src/stores/TwoFactorAuthentificationStore.ts

```diff
@@ -70,7 +72,7 @@ export const useTwoFactorAuthentificationStore: StoreDefinition<
       this.required = tempRequired;
     },
     async get2FAState(personId: string) {
-      this.loading = true;
+      this.loadingInitialState = true;
```

Comment:
Was ist der Unterschied zwischen loading und loadingInitialState?

Reply:
LoadingInitialState wird benutzt, um zu überprüfen, ob 2FA überhaupt angezeigt werden und wie der Token Status aussieht. Loading wird dann im zweiten Schritt verwendet, wenn z.B. Anfragen für neue Softwaretoken gestellt werden.

Reply:
hat sich eh erledigt, weil in main eine Alternativlösung erstellt wurde